Formula/aws-google-auth.rb

```diff
@@ -18,8 +19,8 @@ class AwsGoogleAuth < Formula
   end
 
   depends_on "pillow"
```

Comment:
`pillow` dependency tree has `glib` & `gobject-introspection` even though it does build bindings for multiple Python. Another situation where have occasionally used allowlist.